package.json

```diff
@@ -377,7 +382,7 @@
     "@types/node": "14.0.5",
     "@types/shell-quote": "1.6.2",
     "@types/vscode": "1.43.0",
-    "prettier": "2.0.5",
+    "prettier": "^2.0.5",
```

Comment:
minor: why this change?
For context: we rely on dependabot to send patches for the updates so we prefer to use fixed versions in the `package.json`

Reply:
I thought it was added by `yarn format`. 
I fixed it now.

Reply:
thanks :)